### PR TITLE
User

### PR DESCRIPTION
**Pull request by VanyaRastogi**
**Please confirm that your submission follows all the [rules](https://github.com/Metroxe/one-html-page-challenge/blob/master/README.md#rules)...**
- [x] You wrote all of your code in 1 single HTML file.
- [x] Your file is less than 1mb.
- [x] You didn't import any external files (e.g. images, stylesheets or js files).
- [x] There are no incoming or outgoing network requests.

### CODE REVIEW

File: entries/Shoping-website.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
```
Comment (Metroxe) [inline entries/Shoping-website.html:7]: You cannot use external sources outside of your HTML, please remove this css stylesheet

File: entries/Shoping-website.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+          </a>
+          <div class="mt-4">
+            <h3 class="text-gray-500 text-xs tracking-widest title-font mb-1">AKS</h3>
+            <h2 class="text-gray-900 title-font text-lg font-medium">Traditional Saree</h2>
+            <p class="mt-1">$21.15</p>
+          </div>
+        </div>
+        <div class="lg:w-1/4 md:w-1/2 p-4 w-full">
+          <a class="block relative h-48 rounded overflow-hidden">
+            <img alt="ecommerce" class="object-cover object-center w-full h-full block" src="https://unsplash.com/s/photos/saree">
```
Comment (Metroxe) [inline entries/Shoping-website.html:116]: No external files

File: entries/Shoping-website.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+          </a>
+          <div class="mt-4">
+            <h3 class="text-gray-500 text-xs tracking-widest title-font mb-1">Saree mall</h3>
+            <h2 class="text-gray-900 title-font text-lg font-medium">Traditional Saree</h2>
+            <p class="mt-1">$12.00</p>
+          </div>
+        </div>
+        <div class="lg:w-1/4 md:w-1/2 p-4 w-full">
+          <a class="block relative h-48 rounded overflow-hidden">
+            <img alt="ecommerce" class="object-cover object-center w-full h-full block" src="https://unsplash.com/photos/BtkHK1vhg1E">
```
Comment (Metroxe) [inline entries/Shoping-website.html:126]: again, no external files

File: entries/Shoping-website.html [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+            <h2 class="text-gray-900 title-font text-lg font-medium">Traditional Saree</h2>
+            <p class="mt-1">$18.40</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="text-gray-600 body-font relative">
+    <div class="absolute inset-0 bg-gray-300">
+      <iframe width="100%" height="100%" frameborder="0" marginheight="0" marginwidth="0" title="map" scrolling="no" src="https://maps.google.com/maps?width=100%&amp;height=600&amp;hl=en&amp;q=%C4%B0zmir+(My%20Business%20Name)&amp;ie=UTF8&amp;t=&amp;z=14&amp;iwloc=B&amp;output=embed" style="filter: grayscale(1) contrast(1.2) opacity(0.4);"></iframe>
```
Comment (Metroxe) [inline entries/Shoping-website.html:139]: Literally loading a page from outside the HTML file

**Review by Metroxe — CHANGES_REQUESTED** [review]

File: entries.js [hunk 5]
```diff
@@ -497,4 +497,11 @@ const entries = [
 		author: "Rajiv Sahal",
 		github: "Ryukemeister"
 	}
+	{
```
Comment (Metroxe) [inline entries.js:500]: missing "," causing error in entries.js

**Review by Metroxe — CHANGES_REQUESTED** [review]

File: entries/Shoping-website.html (hunk 1 above)
Comment (VanyaRastogi) [reply entries/Shoping-website.html:7]: ok

**Pull request closed without merging**